diff -p -up desktop-data-model-1.2.5/engine-dbus/hippo-dbus-server.c~ desktop-data-model-1.2.5/engine-dbus/hippo-dbus-server.c
--- desktop-data-model-1.2.5/engine-dbus/hippo-dbus-server.c~	2008-06-16 16:41:18.000000000 +0200
+++ desktop-data-model-1.2.5/engine-dbus/hippo-dbus-server.c	2009-01-02 13:43:45.000000000 +0100
@@ -1730,27 +1730,27 @@ handle_muine_song_changed(HippoDBus   *d
         if (value != NULL && value[1] != '\0' && value[2] != '\0') {
             value += 2;
             if (g_strncasecmp(fields[j], "artist", 6) == 0) {
-                g_debug(fields[j]);
+                g_debug("%s", fields[j]);
                 keys[i] = g_strdup("artist");
                 values[i] = g_strdup(value);
                 ++i;
             } else if (g_strncasecmp(fields[j], "title", 5) == 0) {
-                g_debug(fields[j]);
+                g_debug("%s", fields[j]);
                 keys[i] = g_strdup("name");
                 values[i] = g_strdup(value);
                 ++i;
             } else if (g_strncasecmp(fields[j], "album", 5) == 0) {
-                g_debug(fields[j]);
+                g_debug("%s", fields[j]);
                 keys[i] = g_strdup("album");
                 values[i] = g_strdup(value);
                 ++i;
             } else if (g_strncasecmp(fields[j], "duration", 8) == 0) {
-                g_debug(fields[j]);
+                g_debug("%s", fields[j]);
                 keys[i] = g_strdup("duration");
                 values[i] = g_strdup(value);
                 ++i;
             } else if (g_strncasecmp(fields[j], "track_number", 12) == 0) {
-                g_debug(fields[j]);
+                g_debug("%s", fields[j]);
                 keys[i] = g_strdup("track-number");
                 values[i] = g_strdup(value);
                 ++i;
